dwm.cpp

```diff
@@ -1473,6 +1478,47 @@ setlayout(const Arg *arg)
 		drawbar(selmon);
 }
 
+void layoutmenu(const Arg *arg)
+{
+	/* How many symbols can one layout name fit including the number */
+	int maxlayoutsymbolsize = 10;
+
+	int layoutcount = (sizeof(layouts)/sizeof(*layouts));
+	char layoutlist[layoutcount * maxlayoutsymbolsize];
+	sprintf(layoutlist, "1. %s", layouts[0].symbol);
```

Comment:
why is this needed vs just generating the entire string via the loop

Reply:
Because the lines in the loop need \n newline char in the beginning of the line and the first line is an exception